x-pack/plugins/lens/public/datatable_visualization/components/toolbar.tsx

```diff
@@ -49,6 +40,29 @@ export function DataTableToolbar(props: VisualizationToolbarProps<DatatableVisua
     [state, setState]
   );
 
+  const onChangeHeaderHeight = useCallback(
```

Comment:
nit: it would be nice to refactor these two functions with those above, but it is not urgent.